lib/Controller/Folder.php

```diff
@@ -2,7 +2,7 @@
 /*
  * Copyright (C) 2023 Xibo Signage Ltd
  *
- * Xibo - Digital Signage - https://xibosignage.com
+ * Xibo - Digital Signage - http://www.xibo.org.uk
```

Comment:
I'm not sure if we want to change this 🤔 

Given the copyright update in here and in Handlers.php , perhaps it's not quite correctly configured?